pkg/operator/ceph/cluster/osd/deviceSet.go

```diff
@@ -57,6 +57,7 @@ func (c *Cluster) prepareStorageClassDeviceSets(config *provisionConfig) []rooka
 					},
 					Portable:            storageClassDeviceSet.Portable,
 					TuneSlowDeviceClass: storageClassDeviceSet.TuneSlowDeviceClass,
+					CrushDeviceClass:    pvcTemplate.Annotations["crushDeviceClass"],
```

Comment:
If there are no annotations, this will just be the empty string? I think the integration tests will hit this code path now anyway, just checking.

Reply:
Or do we need to validate the value? Does Ceph require certain values such as sdd, hdd, or hybrid? Or can they specify `foo` if they want?

Reply:
Any value works.